docs/user-guide/ignore-code.md

```diff
@@ -63,7 +63,7 @@ Stylelint supports complex, overlapping disabling & enabling patterns:
 ```
 
 > [!WARNING]
-> Configuration commands in non-standard syntax comments (e.g. `// stylelint-disable`) are only minimally supported.  
```

Comment:
> The rules regarding line breaks in quotes is the same as outside of quotes. If you want to have a single newline, you need to either add two spaces after the preceding line, or add a <br> . 

This change seems out of scope.

Reply:
It seems that auto-formatting was applied. Thanks for checking!

[chore: revert spaces](https://github.com/stylelint/stylelint/pull/8081/commits/170f8e3ab723b1cf5e7a93fac0f5b26127638fdb)